crates/hyperdrive-test-utils/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+#![allow(clippy::comparison_chain)]
```

Comment:
ooo what's this?

Reply:
This is left over from before. It prevents clippy from complaining about using `if` statements in comparison chains that could be rewritten with `match` or `cmp`, but I just rant lint without it locally and it passed, so I don't think it's needed anymore 👍

Reply:
oh yeah, Ive seen those warnings, they're annoying.